File: Samples/C#.Net/CSharpSDKTest/IAccountsDlg.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using SalesMatePlusLib;   

namespace CSharpSDKTest
{
    public partial class IAccountsDlg : Form
    {
        public IAccountsDlg()
        {
            InitializeComponent();
        }

        private void RefundToCustomerbtn_Click(object sender, EventArgs e)
        {
            IAccounts m_Accounts;

            m_Accounts = new AccountsClass();

            object VarCusId = 0;

            m_Accounts.InvokeRefundToCustomerDlg(VarCusId);
        }

        private void AcceptPaymentbtn_Click(object sender, EventArgs e)
        {
            IAccounts m_Accounts;

            m_Accounts = new AccountsClass();

            object VarCusId = 0;

            m_Accounts.InvokeAcceptPaymentDialog(VarCusId);
        }

        private void Adbtn_Click(object sender, EventArgs e)
        {
            IAccounts m_Accounts;

            m_Accounts = new AccountsClass();

            Adlbl.Visible = false;

            string strShpBal;

            double num;

            strShpBal = AdTxt.Text;

            bool isnum=double.TryParse(strShpBal,out num);

            if (!isnum)
            {
                Adlbl.Visible=true;
            }

            else if (strShpBal == "")
            {
                Adlbl.Visible = true;
            }
            else
            {
                m_Accounts.AddToShopBalance(double.Parse(strShpBal));

                MessageBox.Show("Added to the shop balance");
            }
                
         }

        private void Sbtractbtn_Click(object sender, EventArgs e)
        {
            IAccounts m_Accounts;

            m_Accounts = new AccountsClass();

            Sublbl.Visible = false;

            string strShpBal;

            double num;

            strShpBal = SubtractTxt.Text;

            bool isnum = double.TryParse(strShpBal, out num);

            if (!isnum)
            {
                Sublbl.Visible = true;
            }

            else if (strShpBal == "")
            {
                Sublbl.Visible = true;
            }
            else
            {
                m_Accounts.SubstractFromShopBalance(double.Parse(strShpBal));

                MessageBox.Show("Subtracted from the shop balance");
            }

        }

     }
}

File: Samples/VB.Net/VBDotNet/IAccountsDlg.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports SalesMatePlusLib

Namespace CSharpSDKTest
	Partial Public Class IAccountsDlg
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub RefundToCustomerbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles RefundToCustomerbtn.Click
			Dim m_Accounts As IAccounts

			m_Accounts = New AccountsClass()

			Dim VarCusId As Object = 0

			m_Accounts.InvokeRefundToCustomerDlg(VarCusId)
		End Sub

		Private Sub AcceptPaymentbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles AcceptPaymentbtn.Click
			Dim m_Accounts As IAccounts

			m_Accounts = New AccountsClass()

			Dim VarCusId As Object = 0

			m_Accounts.InvokeAcceptPaymentDialog(VarCusId)
		End Sub

		Private Sub Adbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles Adbtn.Click
			Dim m_Accounts As IAccounts

			m_Accounts = New AccountsClass()

			Adlbl.Visible = False

			Dim strShpBal As String

			Dim num As Double

			strShpBal = AdTxt.Text

			Dim isnum As Boolean=Double.TryParse(strShpBal,num)

			If (Not isnum) Then
				Adlbl.Visible=True

			ElseIf strShpBal = "" Then
				Adlbl.Visible = True
			Else
				m_Accounts.AddToShopBalance(Double.Parse(strShpBal))

				MessageBox.Show("Added to the shop balance")
			End If

		End Sub

		Private Sub Sbtractbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles Sbtractbtn.Click
			Dim m_Accounts As IAccounts

			m_Accounts = New AccountsClass()

			Sublbl.Visible = False

			Dim strShpBal As String

			Dim num As Double

			strShpBal = SubtractTxt.Text

			Dim isnum As Boolean = Double.TryParse(strShpBal, num)

			If (Not isnum) Then
				Sublbl.Visible = True

			ElseIf strShpBal = "" Then
				Sublbl.Visible = True
			Else
				m_Accounts.SubstractFromShopBalance(Double.Parse(strShpBal))

				MessageBox.Show("Subtracted from the shop balance")
			End If

		End Sub

	End Class
End Namespace